Author: Jan Christoph Nordholz <hesso@pool.math.tu-berlin.de>
Description: Fix the references to configuration and pipe paths to match the locations Debian uses.
Forwarded: not-needed

--- a/doc/screen.1
+++ b/doc/screen.1
@@ -103,7 +103,7 @@
 the invoking shell to the application (emacs in this case), because it is
 forked from the parent screen process, not from the invoking shell.
 .PP
-If \*Q/etc/utmp\*U is writable by
+If \*Q/run/utmp\*U is writable by
 .IR screen ,
 an appropriate record will be written to this file for each window, and
 removed when the window is terminated.
@@ -232,7 +232,7 @@
 The use of this option is discouraged.
 .TP 5
 .BR \-l " and " \-ln
-turns login mode on or off (for /etc/utmp updating).
+turns login mode on or off (for /run/utmp updating).
 This can also be defined through the \*Qdeflogin\*U .screenrc command.
 .TP 5
 .BR \-ls " [" \fImatch ]
@@ -769,7 +769,7 @@
 
 .SH CUSTOMIZATION
 The \*Qsocket directory\*U defaults either to $HOME/.screen or simply to
-/tmp/screens or preferably to /usr/local/screens chosen at compile-time. If
+/tmp/screens or preferably to /run/screen chosen at compile-time. If
 .I screen
 is installed setuid-root, then the administrator
 should compile
@@ -782,7 +782,7 @@
 When
 .I screen
 is invoked, it executes initialization commands from the files
-\*Q/usr/local/etc/screenrc\*U and
+\*Q/etc/screenrc\*U and
 \*Q.screenrc\*U in the user's home directory. These are the \*Qprogrammer's
 defaults\*U that can be overridden in the following ways: for the
 global screenrc file 
@@ -2050,7 +2050,7 @@
 The echo command may be used to annoy 
 .I screen
 users with a 'message of the
-day'. Typically installed in a global /local/etc/screenrc. 
+day'. Typically installed in a global /etc/screenrc. 
 The option \*Q\-n\*U may be used to suppress the line feed.
 See also \*Qsleep\*U.
 Echo is also useful for online checking of environment variables.
@@ -5196,14 +5196,14 @@
 .I screen
 distribution package for private and global initialization files.
 .IP $SYSSCREENRC 
-.IP /usr/local/etc/screenrc
+.IP /etc/screenrc
 .I screen
 initialization commands
 .IP $SCREENRC
 .IP $HOME/.screenrc
-Read in after /usr/local/etc/screenrc
+Read in after /etc/screenrc
 .IP $SCREENDIR/S\-<login>
-.IP /local/screens/S\-<login>
+.IP /run/screen/S\-<login>
 Socket directories (default)
 .IP /usr/tmp/screens/S\-<login>
 Alternate socket directories.
@@ -5222,7 +5222,7 @@
 or
 .IP /etc/termcap
 Terminal capability databases
-.IP /etc/utmp
+.IP /run/utmp
 Login records
 .IP $LOCKPRG
 Program that locks a terminal.
@@ -5349,9 +5349,9 @@
 must be installed as set-uid with owner root on most systems in order
 to be able to correctly change the owner of the tty device file for
 each window.
-Special permission may also be required to write the file \*Q/etc/utmp\*U.
+Special permission may also be required to write the file \*Q/run/utmp\*U.
 .IP \(bu
-Entries in \*Q/etc/utmp\*U are not removed when
+Entries in \*Q/run/utmp\*U are not removed when
 .I screen
 is killed with SIGKILL.
 This will cause some programs (like "w" or "rwho")
--- a/doc/screen.texinfo
+++ b/doc/screen.texinfo
@@ -185,7 +185,7 @@
 the invoking shell to the application (emacs in this case), because it is
 forked from the parent screen process, not from the invoking shell.
 
-If @file{/etc/utmp} is writable by @code{screen}, an appropriate record
+If @file{/run/utmp} is writable by @code{screen}, an appropriate record
 will be written to this file for each window, and removed when the
 window is closed.  This is useful for working with @code{talk},
 @code{script}, @code{shutdown}, @code{rsend}, @code{sccs} and other
@@ -314,7 +314,7 @@
 
 @item -l
 @itemx -ln
-Turn login mode on or off (for @file{/etc/utmp} updating).  This option
+Turn login mode on or off (for @file{/run/utmp} updating).  This option
 is equivalent to the @code{deflogin} command (@pxref{Login}).
 
 @item -ls [@var{match}]
@@ -497,7 +497,7 @@
 @cindex screenrc
 When @code{screen} is invoked, it executes initialization commands from
 the files @file{.screenrc} in the user's home directory and
-@file{/usr/local/etc/screenrc}.  These defaults can be overridden in the 
+@file{/etc/screenrc}.  These defaults can be overridden in the 
 following ways:
 For the global screenrc file @code{screen} searches for the environment
 variable @code{$SYSSCREENRC} (this override feature may be disabled at
@@ -1089,7 +1089,7 @@
 @item logfile @var{filename}
 Place where to collect logfiles.  @xref{Log}.
 @item login [@var{state}]
-Log the window in @file{/etc/utmp}.  @xref{Login}.
+Log the window in @file{/run/utmp}.  @xref{Login}.
 @item logtstamp [@var{state}]
 Configure logfile time-stamps.  @xref{Log}.
 @item mapdefault
@@ -2405,7 +2405,7 @@
 * Naming Windows::		Control the name of the window
 * Console::			See the host's console messages
 * Kill::                        Destroy an unwanted window
-* Login::                       Control @file{/etc/utmp} logging
+* Login::                       Control @file{/run/utmp} logging
 * Mode::                        Control the file mode of the pty
 * Monitor::                     Watch for activity or inactivity in a window
 * Windows::			List the active windows
@@ -2614,7 +2614,7 @@
 @kindex L
 @deffn Command login [state]
 (@kbd{C-a L})@*
-Adds or removes the entry in @file{/etc/utmp} for the current window.
+Adds or removes the entry in @file{/run/utmp} for the current window.
 This controls whether or not the window is @dfn{logged in}.  In addition
 to this toggle, it is convenient to have ``log in'' and ``log out''
 keys.  For instance, @code{bind I login on} and @code{bind O 
@@ -5802,17 +5802,17 @@
 global initialization files.
 
 @item @code{$SYSSCREENRC}
-@itemx /local/etc/screenrc
+@itemx /etc/screenrc
 @code{screen} initialization commands
 
 @item @code{$SCREENRC}
 @itemx @code{$HOME}/.iscreenrc
 @itemx @code{$HOME}/.screenrc
-Read in after /local/etc/screenrc
+Read in after /etc/screenrc
 
 @item @code{$SCREENDIR}/S-@var{login}
 
-@item /local/screens/S-@var{login}
+@item /run/screen/S-@var{login}
 Socket directories (default)
 
 @item /usr/tmp/screens/S-@var{login}
@@ -5835,7 +5835,7 @@
 @itemx /etc/termcap
 Terminal capability databases
 
-@item /etc/utmp
+@item /run/utmp
 Login records
 
 @item @code{$LOCKPRG}
@@ -5947,10 +5947,10 @@
 in order to be able to
 correctly change the owner of the tty device file for each window.
 Special permission may also be required to write the file
-@file{/etc/utmp}.
+@file{/run/utmp}.
 
 @item
-Entries in @file{/etc/utmp} are not removed when @code{screen} is killed
+Entries in @file{/run/utmp} are not removed when @code{screen} is killed
 with SIGKILL.  This will cause some programs (like "w" or "rwho") to
 advertise that a user is logged on who really isn't.
 
@@ -6030,7 +6030,7 @@
 @cindex socket directory
 
 The socket directory defaults either to @file{$HOME/.screen} or simply to 
-@file{/tmp/screens} or preferably to @file{/usr/local/screens} chosen at 
+@file{/tmp/screens} or preferably to @file{/run/screen} chosen at 
 compile-time. If @code{screen} is installed
 setuid root, then the administrator should compile screen with an
 adequate (not NFS mounted) @code{SOCKDIR}. If @code{screen} is not
